engine/src/contract_methods/connector/deposit_event.rs

```diff
@@ -18,9 +18,16 @@ pub type EventParams = Vec<EventParam>;
 #[derive(BorshSerialize, BorshDeserialize)]
 #[cfg_attr(not(target_arch = "wasm32"), derive(Debug, PartialEq, Eq))]
 pub struct FtTransferMessageData {
-    pub relayer: AccountId,
     pub recipient: Address,
-    pub fee: Fee,
+    #[deprecated]
+    pub fee: Option<FtTransferFee>,
```

Comment:
Why do we still need this field? Saving backward compatibility?

Reply:
Yes, just to save the backward compatibility and make less aggressive changes.
Removing this field and keeping the backward compatibility parse logic is possible, but more cleanup and test adjustments will be needed. Let me know if you think it is better to go that way.